src/main/resources/META-INF/configurations/org.jahia.modules.jcontent.cfg

```diff
@@ -11,3 +11,5 @@ createChildrenDirectButtons.limit = 5
 # forceDateFormat overrides the browser locale format of date in content editor
 # Possible values are: DD/MM/YYYY, MM/DD/YYYY
 # forceDateFormat=DD/MM/YYYY
+
+contentEditorQuickActionsButtons.limit=5
```

Comment:
I would actually just leave this commented out and with explanation of what this does (similar to what we have for forceDateFormat above).